### ISSUE TITLE
Incorrect construction of RecordException

### ISSUE THREAD

**Issue by IvDmNe** [daohu527/cyber_record#66]
In case of broken .record file there is an attempt of creating RecordException object:

https://github.com/daohu527/cyber_record/blob/83656db63ab90be1e6104a12de07fef6fd3eb6eb/cyber_record/reader.py#L268

but it has overrided __init__ function without accepting additional arguments:
https://github.com/daohu527/cyber_record/blob/83656db63ab90be1e6104a12de07fef6fd3eb6eb/cyber_record/record_exception.py#L19

I suggest, that RecordException class should be left empty:
```python
class RecordException(Exception):
    pass
```

**Comment by daohu527** [comment]
@IvDmNe Yes, you are right, I want to create a custom exception to prompt the error message. You can submit a PR to fix it!
